Formula/uade.rb

```diff
@@ -30,45 +31,30 @@ class Uade < Formula
     sha256 x86_64_linux:   "4e9995eda253cbfa248aa486080c04cca295f348e1682d2c10b7ec45a967d46f"
   end
 
-  depends_on "pkg-config" => :build
-  depends_on "libao"
+  head do
+    url "https://gitlab.com/uade-music-player/uade.git", branch: "master"
 
-  resource "bencode-tools" do
-    url "https://github.com/heikkiorsila/bencode-tools.git", branch: "master"
+    resource "bencode-tools" do
+      url "https://gitlab.com/heikkiorsila/bencodetools.git", branch: "master"
+    end
   end
 
+  depends_on "pkg-config" => :build
+  depends_on "libao"
+
   def install
-    if build.head?
-      resource("bencode-tools").stage do
-        system "./configure", "--prefix=#{prefix}", "--without-python"
-        system "make"
-        system "make", "install"
-      end
+    resource("bencode-tools").stage do
+      system "./configure", "--prefix=#{prefix}", "--without-python"
+      system "make"
+      system "make", "install"
     end
 
-    system "./configure", "--prefix=#{prefix}"
+    system "./configure", "--prefix=#{prefix}",
+           "--without-write-audio"
     system "make", "install"
   end
-end
-
-__END__
-diff --git a/configure b/configure
-index 05bfa9b..a73608e 100755
---- a/configure
-+++ b/configure
-@@ -399,13 +399,13 @@ if test -n "$sharedir"; then
-     uadedatadir="$sharedir"
- fi
 
--$NATIVECC -v 2>/dev/null >/dev/null
-+$NATIVECC --version 2>/dev/null >/dev/null
- if test "$?" != "0"; then
-     echo Native CC "$NATIVECC" not found, please install a C compiler
-     exit 1
- fi
-
--$TARGETCC -v 2>/dev/null >/dev/null
-+$TARGETCC --version 2>/dev/null >/dev/null
- if test "$?" != "0"; then
-     echo Target CC "$TARGETCC" not found, please install a C compiler
-     exit 1
+  test do
+    assert_equal "uade123 #{version}", shell_output("#{bin}/uade123 --version").chomp
```

Comment:
Is there no other test we can do?

Reply:
Unfortunately it'd be tricky. This is a music player, which is intended to be used interactively. We'd need a sample music file to use its "real" behaviour, and if we tested, say, rendering the output to a file, validating that that output is *correct* is kind of impossible.

I could see if there's a freely-licensed music file to use as a sample and test if it renders anything at all though.

Reply:
There's mp3 files in the homebrew test files

Reply:
This isn't an MP3 player - it plays tracker modules for the Amiga. We don't currently have any of those.

Reply:
What if you give it a file it can't read? Will it give an error we can check?

Reply:
That's doable. Pushed one that asserts that it returns the expected error status and the expected error message.